pkg/kn/commands/testing_helper.go

```diff
@@ -93,33 +93,44 @@ func CreateDynamicTestKnCommand(cmd *cobra.Command, knParams *KnParams, objects
 
 }
 
-type StdoutCapture struct {
-	r, w *os.File
-	t    *testing.T
+type OutputCapture struct {
```

Comment:
Perhaps this `testing_helper.go` should move to `lib/test` to make it useable in `client-contrib` plugins?

Reply:
Good idea, going to move it over there.

Reply:
I just moved the `OutputCapture` to `lib/test`. 

I think the other parts are more for internally testing `KnCommands` that is specific of how `kn` organizes it commands (also there might be some change here, too).